src/components/buttons/_buttons.scss

```diff
@@ -11,6 +11,12 @@ $largeButtonSize: componentSize(l);
   }
 }
 
+@mixin iconColor($color) {
```

Comment:
@katarzynatobis `buttonIconColor` ? `iconColor` is already defined mixing and this can be easily confused.

Reply:
👍🏻